### PR TITLE
Replace aiozk with Kazoo in asab-zookeeper

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

**Comment by mithunbharadwaj** [comment]
@ateska Ales not yet! I will notify you.
I will extend the example too

**Comment by mithunbharadwaj** [comment]
@ateska I set up a call with UI team next week mostly we will test on Monday itself. Its fair to say only when they say its okay ,lets go ahead merge it

**Comment by mithunbharadwaj** [comment]
**CALLS WORKING:**
**asab-config:**

GET Config
SET Config
DELETE Config
GET Configs

GET Type
GET Types

File: asab/api/service.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 asab.Config.add_defaults({
 	"general": {
-		"manifest": "",
+		"manifest": "app/MANIFEST.json",
```
Comment (byewokko) [inline asab/api/service.py:20]: Is this indeed correct? It doesn't seem compatible with the default value handling that you do here: https://github.com/TeskaLabs/asab/blob/Refactor/Use-Kazoo-asab-zookeeper/asab/api/service.py#L34

File: examples/zookeeper.py [hunk 2]
```diff
@@ -17,9 +17,8 @@ def __init__(self):
 
 		# Advertise self thru ZooKeeper
 		zksvc = self.get_service("asab.ZooKeeperService")
+		zksvc.DefaultContainer
```
Comment (byewokko) [inline examples/zookeeper.py:20]: This looks unfinished. You could add an example of writing/reading zookeeper data here. Or at least a TODO note.

Comment (mithunbharadwaj) [reply examples/zookeeper.py:20]: I am working on this now!

File: asab/api/service.py (hunk 1 above)
Comment (ateska) [reply asab/api/service.py:20]: "" should trigger all the "magic" 

`"manifest": ""` is correct

**Review by ateska — CHANGES_REQUESTED** [review]

File: asab/api/service.py [hunk 3]
```diff
@@ -32,10 +32,8 @@ def __init__(self, app, service_name="asab.ApiService"):
 		self.AttentionRequired = {}  # dict of errors found.
 		path = asab.Config.get("general", "manifest")
 		if len(path) == 0:
-			if os.path.isfile("/MANIFEST.json"):
-				path = "/MANIFEST.json"
-			elif os.path.isfile("MANIFEST.json"):
-				path = "MANIFEST.json"
+			if os.path.isfile("app/MANIFEST.json"):
```
[the remark this answers is not in the document]
Comment (ateska) [reply asab/api/service.py:35]: Yes, that's a correct remark.

File: asab/api/service.py [hunk 4]
```diff
@@ -32,10 +32,10 @@ def __init__(self, app, service_name="asab.ApiService"):
 		self.AttentionRequired = {}  # dict of errors found.
 		path = asab.Config.get("general", "manifest")
 		if len(path) == 0:
-			if os.path.isfile("/MANIFEST.json"):
-				path = "/MANIFEST.json"
-			elif os.path.isfile("MANIFEST.json"):
-				path = "MANIFEST.json"
+			if os.path.isfile("/app/MANIFEST.json"):
+				path = "/app/MANIFEST.json"
+			elif os.path.isfile("app/MANIFEST.json"):
```
Comment (mithunbharadwaj) [inline asab/api/service.py:37]: @ateska is this correct?
If you agree I will go ahead and Merge

**Pull request merged**